live-site/components/judging-section/judging-schedule-welcome/JudgingScheduleWelcome.tsx

```diff
@@ -24,14 +50,23 @@ const JudgingScheduleWelcome: React.FC<JudgingScheduleWelcomeProps> = ({
           Welcome, {schedulePersonType}!
         </StyledWelcomePerson>
         <StyledJudgesDropdownContainer>
-          <StyledJudgesDropdownWrapper id="position-filter">
+          <StyledJudgesDropdownWrapper
+            id="position-filter"
+            onChange={(e): void => {
+              setPersonSelected(e.target.value);
+              setCurRows(getRowsAndRoomForJudge(e.target.value, headers).rows)
+              setRoom(getRowsAndRoomForJudge(e.target.value, headers).room)
+            }}
+          >
             {personOptions.map((currPerson: string) => (
               <option value={currPerson} key={currPerson}>
                 {currPerson}
               </option>
             ))}
           </StyledJudgesDropdownWrapper>
         </StyledJudgesDropdownContainer>
+        {personSelected != 'Select your name' && <StyledP>{room}</StyledP> }
```

Comment:
Add "Room: " before the actual room and add some padding above + below it so its not so close to the elements around it
<img width="626" alt="Screenshot 2023-02-10 at 12 32 12 AM" src="https://user-images.githubusercontent.com/59738880/218009031-7c39ffa9-ad78-42ea-849b-cb7e6f989b91.png">